lib/modulesync/git.rb

```diff
@@ -95,12 +105,18 @@ def self.tag(repo, version, tag_pattern)
       repo.push('origin', tag)
     end
 
+    def self.checkout_branch(repo, branch)
```

Comment:
The name is quite similar to ```switch_branch```. Maybe this should be a private method to clearly indicate it's a helper?